_rules/focusable-no-keyboard-trap-standard-nav-a1b64e.md

```diff
@@ -29,12 +29,13 @@ This rule applies to any [HTML or SVG element][] that is [focusable][].
 
 For each target element, focus can cycle to the browser UI by using [standard keyboard navigation](#standard-keyboard-navigation).
 
-**Note:** Cycling back to the browser UI can be done both by moving forward through the tab order and by moving backwards. It is not possible to fulfill this expectation by using browser specific shortcuts to return to the browser UI.
+**Note:** It is not possible to fulfill this expectation by using browser specific shortcuts to return to the browser UI.
```

Comment:
I'm not so sure that things that can only be exited with shift+tab should pass this SC. I think we need to get clarity on that. This note shouldn't have been a note. Knowing whether its backward, forward, or both needs to be in the expectation, but then by taking this out of the note without changing the expectation we're making the rule less clear.